ERCS/erc-0000.md

```diff
@@ -0,0 +1,97 @@
+---
+eip: 0000
+title: Security Layers
```

Comment:
Is there a more descriptive title you could use?  Composable runtime security checks

Reply:
Changed it to `Composable Security Middleware Hooks` , hope this sounds more self-explanatory